### PR TITLE
Glide instead of Godep

### PR DESCRIPTION
**Pull request by katarzyna-z**
Summary of changes:
- Replaced godep with glide

How to verify it:
- execute make

Testing done:
- small tests,
- manual tests

### CODE REVIEW

File: glide.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  - control/plugin
+  - core
+  - pkg/ctree
+  - pkg/schedule
+  - scheduler/wmap
+  - control/plugin/cpolicy
+  - core/ctypes
+- package: github.com/robfig/cron
+  version: 32d9c273155a0506d27cf73dd1246e86a470997e
+- package: gopkg.in/yaml.v2
```
Comment (marcin-krolik) [inline glide.yaml]: I think it is `snap` dependency, not plugins. Remove if that's the case

File: glide.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  version: dbbb1ae3ef7282ac02bd6e14ef44d93fb7c9e85b
+  subpackages:
+  - control/plugin
+  - core
+  - pkg/ctree
+  - pkg/schedule
+  - scheduler/wmap
+  - control/plugin/cpolicy
+  - core/ctypes
+- package: github.com/robfig/cron
```
Comment (marcin-krolik) [inline glide.yaml]: This also seems to be dependency of some other package (snap?). Remove if that's the case

Comment (katarzyna-z) [inline glide.yaml]: it is dependency of other packages. I removed it.

File: glide.yaml (hunk 1 above)
Comment (katarzyna-z) [inline glide.yaml]: I also removed this dependency.

**Comment by marcin-krolik** [comment]
LGTM

**Pull request merged**